project/ws/app/src/lib/routes/home/routes/users-view/users-view.component.ts

```diff
@@ -217,6 +218,12 @@ export class UsersViewComponent implements OnInit, OnDestroy {
     reqBody = {
       request: {
         filters: filtreq,
+        facets: [
+          'profileDetails.professionalDetails.group',
```

Comment:
Check if profileDetails exists or not

Reply:
this payload to be send can't add condition here